week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,7 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = students.every(group); // complete this statement
```

Comment:
I'm afraid this code doesn't run :( `group` is not a function, it's an array. The `every()` method needs a function (or the name of a function) as parameter. That function needs to return a boolean based on a certain check. Here this check would be to see if the element of the array on which the method is being called on belongs in the `students` array. 

The interpretation of the task is: check if every element in the group array is included in the students array.

Hope this helps! :)